src/main/java/net/pms/encoders/MEncoderVideo.java

```diff
@@ -251,10 +251,10 @@ public void itemStateChanged(ItemEvent e) {
 					fakemedia.setCodecV("mpeg4");
 					fakemedia.setContainer("matroska");
 					fakemedia.setDuration(45d*60);
-					audio.getAudioProperties().setNumberOfChannels(2);
+					audio.setNumberOfChannels(2);
 					fakemedia.setWidth(1280);
 					fakemedia.setHeight(720);
-					audio.setSampleFrequency("48000");
+					audio.setSampleRate(48000);
```

Comment:
@Sami32 That was my idea as well, but I later discovered that while MediaInfo use "sample rate" DLNA use "sample frequency" so we won't get rid of the confusion whatever we do.